libsolidity/ast/AST_accept.h

```diff
@@ -58,13 +59,19 @@ void PragmaDirective::accept(ASTConstVisitor& _visitor) const
 
 void ImportDirective::accept(ASTVisitor& _visitor)
 {
-	_visitor.visit(*this);
+	if (_visitor.visit(*this))
+		for (SymbolAlias const& symbolAlias: symbolAliases())
+			if (symbolAlias.symbol)
+				symbolAlias.symbol->accept(_visitor);
```

Comment:
I think this is a correct change, but we have to cross-check how this impacts the other code - we essentially have to take a look at all `visit(Identifier` and `endVisit(Identifier` functions to see that this is fine.